APIKitTests/ResponseBodyParserTests.swift

```diff
@@ -37,14 +38,14 @@ class ResponseBodyParserTests: XCTestCase {
 
         case .Failure(let box):
             let error = box.unbox
-            XCTAssertEqual(error.domain, NSCocoaErrorDomain)
-            XCTAssertEqual(error.code, 3840)
+            assert(error.domain, ==, NSCocoaErrorDomain)
```

Comment:
Because `NSCocoaErrorDomain` is `NSString`, `assert<T, U>` or `NSCocoaErrorDomain as String` is required.